public/i18n/qqq.json

```diff
@@ -25,7 +25,8 @@
 	"item-form-error-message-max": "An error message when too many item ids are provided",
 	"item-form-error-message-invalid": "An error message when invalid Item ids are provided",
 	"server-error": "An error message when the server encountered an error retrieving results",
-	"column-property": "Property column on mismatches table",
+	"column-mismatch": "Property column on mismatches table",
```

Comment:
I guess we should change this to say something else? so the people translating will know what this message is about. Though something like "mismatch column on mismatches table" sounds a bit meaningless to me, maybe you have a better idea?

Reply:
I thought this was being changed from "property" to "mismatch" to allow adding them for qualifiers, so leaving the word property there has potential for confusion, imo. Though I agree "Mismatch column etc." is no good.

Reply:
I guess we have to reach an agreement on what is good. Mismatch column could be ok but sounds a bit repetitive